hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -334,6 +334,12 @@ private FlinkOptions() {
       .withDescription("Enables data-skipping allowing queries to leverage indexes to reduce the search space by"
           + "skipping over files");
 
+  public static final ConfigOption<Boolean> READ_CONTINUOUS_PARTITION_PRUNING_ENABLED = ConfigOptions
+      .key("read.continuous.partition-pruning.enabled")
```

Comment:
Can we follow the existing specification and change this configuration to `read.streaming.....`?